### PR TITLE
ELPP-3335 Add redis_cache and redis_sessions parameters

### PR DESCRIPTION
**Pull request by giorgiosironi**
Depends on https://github.com/elifesciences/builder-private/pull/106

### CODE REVIEW

File: salt/pillar/journal.sls [hunk 1]
```diff
@@ -16,6 +16,8 @@ journal:
     web_users: {}
 
     redis: redis://localhost
```
Comment (thewilkybarkid) [inline salt/pillar/journal.sls]: Also deprecated

**Comment by giorgiosironi** [comment]
Build is being throttled...

**Comment by thewilkybarkid** [comment]
:sigh:

**Comment by giorgiosironi** [comment]
Enabling "T2 Unlimited", let's see if it makes a difference

**Comment by giorgiosironi** [comment]
Sort of, it got to the expected failure on critical css.

**Comment by thewilkybarkid** [comment]
`TypeError: node.children.each is not a function` - not seen that before.

**Comment by giorgiosironi** [comment]
Not sure of that but the `stderr` said:
```
[34.229.175.241] out:                   Chromium unexpecedly not opened - crashed? 
[34.229.175.241] out:                   _browserPagesOpen: 1
[34.229.175.241] out:                   url: http://localhost:8080/resources
[34.229.175.241] out:                   astRules: 1431
[34.229.175.241] out:                   Chromium unexpecedly not opened - crashed? 
[34.229.175.241] out:                   _browserPagesOpen: 1
[34.229.175.241] out:                   url: http://localhost:8080/resources
[34.229.175.241] out:                   astRules: 1402
```

**Comment by thewilkybarkid** [comment]
Retrying, will investigate locally if it fails.

**Comment by giorgiosironi** [comment]
Seems consistent:
```
[184.73.23.182] out:           ID: generate-critical-css
[184.73.23.182] out:     Function: cmd.run
[184.73.23.182] out:         Name: node_modules/.bin/gulp critical-css:generate
[184.73.23.182] out:       Result: False
[184.73.23.182] out:      Comment: Command "node_modules/.bin/gulp critical-css:generate" run
[184.73.23.182] out:      Started: 12:18:46.529130
[184.73.23.182] out:     Duration: 3382.67 ms
[184.73.23.182] out:      Changes:   
[184.73.23.182] out:               ----------
[184.73.23.182] out:               pid:
[184.73.23.182] out:                   9326
[184.73.23.182] out:               retcode:
[184.73.23.182] out:                   1
[184.73.23.182] out:               stderr:
[184.73.23.182] out:                   Chromium unexpecedly not opened - crashed? 
[184.73.23.182] out:                   _browserPagesOpen: 1
[184.73.23.182] out:                   url: http://localhost:8080/resources
[184.73.23.182] out:                   astRules: 1431
[184.73.23.182] out:                   Chromium unexpecedly not opened - crashed? 
[184.73.23.182] out:                   _browserPagesOpen: 1
[184.73.23.182] out:                   url: http://localhost:8080/resources
[184.73.23.182] out:                   astRules: 1402
[184.73.23.182] out:               stdout:
[184.73.23.182] out:                   [12:18:48] Using gulpfile /srv/journal/gulpfile.js
[184.73.23.182] out:                   [12:18:48] Starting 'critical-css:clean'...
[184.73.23.182] out:                   [12:18:48] Finished 'critical-css:clean' after 7.33 ms
[184.73.23.182] out:                   [12:18:48] Starting 'critical-css:generate'...
[184.73.23.182] out:                   restarting chrome after crash
[184.73.23.182] out:                   [12:18:49] 'critical-css:generate' errored after 1.42 s
[184.73.23.182] out:                   [12:18:49] TypeError: node.children.each is not a function
[184.73.23.182] out:                       at Object.eval [as StyleSheet] (eval at createWalker (/srv/journal/node_modules/css-tree/lib/walker/create.js:235:29), <anonymous>:5:15)
[184.73.23.182] out:                       at walk (/srv/journal/node_modules/css-tree/lib/walker/create.js:269:39)
[184.73.23.182] out:                       at walkUp (/srv/journal/node_modules/css-tree/lib/walker/create.js:276:13)
[184.73.23.182] out:                       at Object.toPlainObject (/srv/journal/node_modules/css-tree/lib/convertor/create.js:18:13)
[184.73.23.182] out:                       at /srv/journal/node_modules/penthouse/lib/index.js:169:38
[184.73.23.182] out:                       at next (native)
[184.73.23.182] out:                       at step (/srv/journal/node_modules/penthouse/lib/index.js:45:191)
[184.73.23.182] out:                       at /srv/journal/node_modules/penthouse/lib/index.js:45:437
[184.73.23.182] out:                       at /srv/journal/node_modules/penthouse/lib/index.js:45:99
[184.73.23.182] out:                       at generateCriticalCssWrapped (/srv/journal/node_modules/penthouse/lib/index.js:267:18)
[184.73.23.182] out:                       at /srv/journal/node_modules/penthouse/lib/index.js:236:21
[184.73.23.182] out:                       at next (native)
[184.73.23.182] out:                       at step (/srv/journal/node_modules/penthouse/lib/index.js:45:191)
[184.73.23.182] out:                       at /srv/journal/node_modules/penthouse/lib/index.js:45:361
[184.73.23.182] out:                       at process._tickCallback (internal/process/next_tick.js:109:7)
```

**Comment by thewilkybarkid** [comment]
Diving in. Must be unrelated though.

**Comment by giorgiosironi** [comment]
Yes, not related to this PR.

**Comment by thewilkybarkid** [comment]
Can't recreate locally; merging this.

**Pull request merged**